spec/person_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/person'
+
+describe Person do
+  books = [{ title: 'Alfons leker', author: 'A. Andersson' }, { title: 'Bible', author: 'TGAOTU' } ]
```

Comment:
Metrics/LineLength: Line is too long. [100/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)